lib/improver/tests/weighted_blend/test_weighted_blend.py

```diff
@@ -55,7 +57,18 @@ def example_coord_adjust(pnts):
     return pnts[len(pnts)-1]
 
 
-class TestBasicWeightedAverage(IrisTest):
+class Test__repr__(IrisTest):
+
+    """Test the repr method."""
+
+    def test_basic(self):
+        """Test that the __repr__ returns the expected string."""
+        result = str(WeightedBlend('time'))
+        msg = '<WeightedBlend: coord = time>'
+        self.assertEqual(result, msg)
+
+
+class TestWeightedBlend(IrisTest):
```

Comment:
I think this is something that is causing conflicts as I changed this to be Test_process during Technical Debt Day